src/buildCommon.js

```diff
@@ -41,6 +41,23 @@ var makeSymbol = function(value, style, mode, color, classes) {
 };
 
 /**
+ * Makes a strut
+ */
+var makeStrut = function() {
+    return new domTree.symbolNode("", 0.85, 0.35);
```

Comment:
It should be 0.84 and 0.36.  I'll change this to be ```0.7 * metrics.baselineSkip``` and ```0.3 * metrics.baselineSkip``` so that people know where those numbers are coming from.  I guess we could re-use the ```domTree.symbolNode``` because it's not a real DOM node.